libs/src/services/ethContracts/index.js

```diff
@@ -31,6 +33,7 @@ const StakingProxyKey = 'StakingProxy'
 const DelegateManagerRegistryKey = 'DelegateManager'
 const ClaimsManagerProxyKey = 'ClaimsManagerProxy'
 const ClaimDistributionRegistryKey = 'ClaimDistribution'
+const EthRewardsManagerProxyKey = 'EthRewardsManagerProxy'
```

Comment:
does this key work? is it actually registered to the proxy?

Reply:
yup! :)